container/webui/Dockerfile

```
@@ -9,6 +9,7 @@ RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/op
     zypper --gpg-auto-import-keys ref && \
     zypper in -y ca-certificates-mozilla curl && \
     zypper in -y --force-resolution openQA-local-db apache2 hostname which w3m && \
+    zypper in -y --force-resolution ruby2.5 ruby2.5-rubygem-sass &&\
```

This would be in accordance with our normal dependencies:
```suggestion
    zypper in -y --force-resolution 'rubygem(sass)' &&\
```
(Installing a rubygem should pull-in ruby automatically.)